File: client/frontend/src/app/shared/models/item.ts
export class Item {
    id: string; //GUID
    manufacturer: string;
    name: string;
    price: number;
    constructor(id: string, manufacturer: string, name: string, price: number){
        this.id = id;
        this.manufacturer = manufacturer;
        this.name = name;
        this.price = price;
    }
}

File: client/viewmodels/Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace viewmodels
{
    public class Item
    {
        public Guid? Id { get; set; }
        public int Version { get; set; }
        public string? Status { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
        public string? Manufacturer { get; set; }
        public string? Name { get; set; }
        public double Price { get; set; }
    }
}
